python/ray/serve/_private/controller.py

```diff
@@ -619,7 +620,7 @@ def deploy(
 
         return updating
 
-    def deploy_application(self, name: str, deployment_args_list: List[Dict]) -> None:
+    def deploy_application(self, name: str, deployment_args_list_bytes: bytes) -> None:
```

Comment:
please update the doc string correspondingly.

Reply:
The unnecessary proto definition `DeploymentArgsList` has been removed, and the doc string has been updated.